test/topology/conftest.py

```diff
@@ -186,7 +186,7 @@ async def manager(request, manager_internal, record_property, build_mode):
                         )
     test_log = suite_testpy_log.parent / f"{suite_testpy_log.stem}.{test_case_name}.log"
     # this should be consistent with scylla_cluster.py handler name in _before_test method
-    test_py_log_test = suite_testpy_log.parent / f"{test_case_name}.log"
+    test_py_log_test = suite_testpy_log.parent / f"{suite_testpy_log.stem}.log"
```

Comment:
Does this fix the problem of log file not found, that we saw in CI recently?

Reply:
Yes.  I submitted https://github.com/scylladb/scylladb/pull/22388 separately for that

Reply:
@denesb can you please review and consider merging #22388?

Reply:
removed dependency and pulled out of draft mode

Reply:
The extracted PR (its replacement https://github.com/scylladb/scylladb/pull/22415) was merged.